modules/exploits/windows/local/capcom_sys_exec.rb

```diff
@@ -21,7 +21,7 @@ def initialize(info = {})
             This module abuses the Capcom.sys kernel driver's function that allows for an
             arbitrary function to be executed in the kernel from user land. This function
             purposely disables SMEP prior to invoking a function given by the caller.
-            This has been tested on Windows 7, 8.1 and Windows 10 (x64).
+            This has been tested on Windows 7, 8.1, 10 (x64) and Windows 11 (x64) upto build 22000.194.
```

Comment:
Up to a build? Out of interest was there some error past this build number? If so, would be good to add this in to the documentation.

Reply:
The windows 11 after build 22000 contains blacklist that blocks the vulnerable capcom.sys driver from getting loaded. I tried it with various versions but got no success after the build 22000.

Reply:
Alright if we can add some documentation on this to the module's docs file and to the module itself that information is great to have and should be added for future travelers 👍

Reply:
Sure, i'll add those.

Reply:
Added with https://github.com/rapid7/metasploit-framework/pull/17474/commits/d97ccaa3d238ce86974c58272f2847e238bdd3e6